scripts/generate-manufacturer-certs.sh

```diff
@@ -2,16 +2,15 @@
 
 source "$(cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )/cert-utils.sh"
 
-ENV_FILE="/etc/sysconfig/go-fdo-server-manufacturer"
-[ ! -f "${ENV_FILE}" ] || source "${ENV_FILE}"
-
-conf_dir="${MANUFACTURER_CONF_DIR:-/etc/go-fdo-server}"
-
-subj="${MANUFACTURER_SUBJECT:-/C=US/O=FDO/CN=Manufacturer}"
-key="${MANUFACTURER_KEY:-${conf_dir}/manufacturer.key}"
-crt="${MANUFACTURER_CRT:-${conf_dir}/manufacturer.crt}"
+cert_dir="/etc/pki/go-fdo-server"
+subj="/C=US/O=FDO/CN=Manufacturer"
+key="${cert_dir}/manufacturer-example.key"
+crt="${cert_dir}/manufacturer-example.crt"
 
+# Do not overwrite existing cert/key files unless one of the pair is
+# missing
+if [[ ! -f "${key}" || ! -f "${crt}" ]]; then
+  rm -f "${key}" "${crt}"
+fi
 generate_cert "${key}" "${crt}" "${subj}"
-
-"$(cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )/generate-device-ca-certs.sh"
-"$(cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )/generate-owner-certs.sh"
+chmod g+r "${key}"
```

Comment:
The manufacturer private key must belong and it must be readable by the manufacturer user only as it's the only one using it for signing.

The manufacturer cert must belong to the manufacturer user and the group permissions are not important because it's not used by any other service